knowledge-base/window-expand-on-drag.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Window Expands When Dragged
+description: The Blazor TelerikWindow component grows the first time it is dragged. Its right border sticks to the right side of the screen.
+type: troubleshooting
+page_title: Window Expands When Dragged for the First Time
+slug: window-kb-expand-on-drag
+position: 
+tags: window, expand, drag, grid, move
+ticketid: 1522796, 1524154
+res_type: kb
+---
+
+## Description
+
+I have a Window with no fixed width and the `Draggable` property is set to true. Initially the Window is rendered to be as wide as its content.
+When the user starts dragging the component for the first time, it grows and sticks to the right side of the screen.
+
+## Cause\Possible Cause(s)
+
+The Window may expand on drag if it has no fixed width, and its content has no width either.
+
+The Window is an absolutely positioned element. If it has no width, it will expand to enclose its content. If the Window content can expand horizontally without a limit, this can cause the Window to expand until it reaches the viewport boundary.
```

Comment:
```suggestion
The Window is an absolutely positioned element. If it has no width, it will expand to enclose its content. If the Window content can expand horizontally without a limit (like block-level elements do), this can cause the Window to expand until it reaches the viewport boundary.
```